website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Jest 26: Tick Tock
+author: Christoph Nakazawa
+authorURL: http://twitter.com/cpojer
+authorFBID: 100000023028168
+---
+
+# Jest 26: Tick Tock
+
+When we started rebuilding Jest five years ago our goal was to provide a batteries-included zero-configuration test runner that is approachable for beginners, extensible for almost all testing use cases and scalable to large projects. One of the most important releases was [Jest 15](https://jestjs.io/blog/2016/09/01/jest-15) which tied all the bits and pieces together and provided great defaults that allowed people to install and run Jest often without any setup. However, this approach has a big downside as Jest installs a lot of dependencies into your projects that you may not need.
+
+We are now beginning to address this shortcoming and are working on reducing Jest’s install size while keeping it approachable and extensible. We have made the following breaking changes in Jest 26:
+
+<!--truncate-->
+
+* `[expect, jest-mock, pretty-format]` Remove `build-es5` from all packages ([#9945](https://github.com/facebook/jest/pull/9945))
+* `[jest-config, jest-resolve]` Remove support for `browser` field ([#9943](https://github.com/facebook/jest/pull/9943))
+* TypeScript definitions requires a minimum of TypeScript v3.8 ([#9823](https://github.com/facebook/jest/pull/9823))
+
+With these changes combined, Jest 26 is now X MiB smaller than Jest 25.5.4 (Y → Z MiB). While this is a good start, it isn’t enough to make Jest meaningfully leaner. Since most of the changes to reduce Jest’s default setup are breaking, we want to bring the community with us as much as possible. Therefore we will make the following changes in Jest 27 and Jest 28 using a [“Tick-Tock" release process](https://en.wikipedia.org/wiki/Tick%E2%80%93tock_model):
```

Comment:
Based on https://packagephobia.now.sh/result?p=jest@26.0.0-alpha.0:
```suggestion
With these changes combined, Jest 26 is now 2.6 MiB smaller than Jest 25.5.4 (35.7 MiB → 33.1 MiB). While this is a good start, it isn’t enough to make Jest meaningfully leaner. Since most of the changes to reduce Jest’s default setup are breaking, we want to bring the community with us as much as possible. Therefore we will make the following changes in Jest 27 and Jest 28 using a [“Tick-Tock" release process](https://en.wikipedia.org/wiki/Tick%E2%80%93tock_model):
```

Reply:
Can you change the language to use "MB" instead of "MiB" then? They are different things, `packagephobia` uses MB.

Reply:
I'd say let's stick to the numbers you provided based on local install